scripts/remove-duplicate-emails.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Script to remove users with
+ *
+ * to run:
+ * drush --script-path=../scripts/ php-script remove-duplicate-emails.php
+ */
+
+$users = db_query('SELECT * FROM users 
```

Comment:
1. Good call. ❌ 
2. Ooof, yeah 203 seconds. Since it's iterating over a heavily filtered set of UIDs (rather than the whoooooole thing like the other script), I'm not sure saving a UID would help out too much. But on the bright side, we can easily just keep running this script over and over until there's no more dupes.